api/mapping/models.py

```diff
@@ -460,14 +460,11 @@ class ScanReportField(BaseModel):
 
     concept_id = models.IntegerField(
         default=-1,
-        null=True,
```

Comment:
Do we need to remove null and blank? If not, I would leave them here. And, I would add a note saying this field is not used anymore.